### ISSUE TITLE
When trying to send tokens, Rabby returns to homescreen after switching tabs

### ISSUE THREAD

**Issue by vaidab** [RabbyHub/Rabby#1585]
Situation: I have a token address in one tab and the exchange address in another tab.
Rabby doesn't detect the token so I need to add it via copy/paste from tab1 and then switch to the exchange and copy/paste the destination address from tab2.
When I switch tabs and click on the Rabby icon again to paste the exchange address, my send transaction initiated in tab1 is no more. I only get the home screen.
Basically I can't switch tabs with the send transaction open.
My workaround is to copy/paste the addresses in a notes app to have them without switching tabs.
It would be nice for Rabby to remember where I left it when I switched the tabs.

**Comment by vvvvvv1vvvvvv** [comment]
can't understand, can you make a screen record video for easy understand?

**Comment by vvvvvv1vvvvvv** [comment]
should be resolved in the latest version, the Send Token page will cached once you change anything
